tools/test.py

```diff
@@ -176,7 +176,7 @@ def process_product(i, product):
 
 
 ontobee = 'http://www.ontobee.org/browser/rdf.php?o=%s&iri=http://purl.obolibrary.org/obo/'
-ols = 'https://www.ebi.ac.uk/ols/ontologies/%s/terms?iri=http%%3A%%2F%%2Fpurl.obolibrary.org%%2Fobo%%2F'
+ols = 'https://www.ebi.ac.uk/ols4/ontologies/%s/terms/http%%253A%%252F%%252Fpurl.obolibrary.org%%252Fobo%%252F$1'
```

Comment:
The URL encoding of both look somewhat different.. Why? `ols = '253A%%252F%%252Fpurl` vs `3A%%2F%%2Fpurl`.